build.zig

```diff
@@ -32,9 +32,9 @@ pub fn build(b: *std.Build) void {
     // Creates a step for unit testing. This only builds the test executable
     // but does not run it.
     const main_tests = b.addTest(.{
-        .root_source_file = .{ .path = "src/test.zig" },
-        .target = target,
-        .optimize = optimize,
+        .root_source_file = b.path("src/test.zig"),
+        .target = b.host,
```

Comment:
Why `b.host`, instead of target?

```suggestion
        .target = target,
```

Reply:
for some reason i thought it should be like that but when i think about it now it makes no sense because we might want to test on different targets so, my mistake